minor updates (#5)
learn-co-curriculum/phase-1-array-iterator-methods: README.md

## Learning Goals

> - Learn how JavaScript's iterator methods help us

Reviewer: Recognize how Javascript's iterator methods help us.

| Use Case                                                       | Method                |
| -------------------------------------------------------------- | --------------------- |

> | Execute a provided method for each element                     | `forEach()`           |

Author: ```suggestion
| Execute a provided callback on each element                    | `forEach()`           |
```
